.github/workflows/joss.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         cd doc
         cp -r ../../doc/joss ./
         cd joss
-        pandoc paper.md -o paper.pdf --bibliography ./paper.bib --latex-engine=xelatex
+        pandoc paper.md -o paper.pdf --bibliography ./paper.bib --pdf-engine=xelatex
```

Comment:
is it due to the pandoc update?

Reply:
it seems like it, indeed with recent versions there is only mention of `--pdf-engine` in the manual